deploy/clean_up_bq_test_data.sh

```diff
@@ -2,6 +2,6 @@
 export DJANGO_SETTINGS_MODULE=openprescribing.settings.test
 LOGFILE=$(mktemp -t clean_up_bq_test_data-$(date +%Y-%m-%d)-XXXX.log)
 
-. /webapps/openprescribing/.venv/bin/activate
```

Comment:
This is actually the opposite of what I was thinking. My suggestion was to replace the existing virtualenv with a symlink to `.venv35` so that we could switch virtualenvs _without_ needing to update references everywhere.

Reply:
That makes much more sense.  I'll back out this change and set up virtualenv symlinking on staging.